app/views/overview/_service-instance-row.html

```diff
@@ -152,6 +152,14 @@ <h3>
                 expandable="true"
                 linkify="true">
               </truncate-long-text></p>
+          </div>
+        </div>
+        <mobile-service-clients ng-if="row.isMobileService && row.instanceStatus === 'ready'"
```

Comment:
create 2 filters for row `isMobileService` and `isInstanceReady` then `ng-if="row | isMobileService | isInstanceReady"`

Reply:
* Should this mobile clients section show up even if there are no mobile clients in the namespace?

Reply:
I've done this now in mine so ignore